rs/sns/governance/canister/governance.did

```diff
@@ -21,6 +21,135 @@ type Action = variant {
   ExecuteGenericNervousSystemFunction : ExecuteGenericNervousSystemFunction;
   ManageLedgerParameters : ManageLedgerParameters;
   Motion : Motion;
+  RegisterExtension : RegisterExtension;
+  DeregisterExtension : DeregisterExtension;
+  ExecuteExtensionFunction : ExecuteExtensionFunction;
+};
+
+type ExistingCanister = record {
+  canister_id : opt principal;
+  wasm_module_hash : opt blob;
+};
+
+type NewCanister = record {
+  chunked_canister_wasm : opt ChunkedCanisterWasm;
+};
+
+type Installation = variant {
+  existing_canister : ExistingCanister;
+  new_canister : NewCanister;
+};
+
+type Icrc2Token = variant {
```

Comment:
since variants are hard to change... 

Are there other flavors of tokens we want to support, such as ckUSDC or ckETH?   Is there a simple enough abstraction where these could be added here?